lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp

```diff
@@ -317,9 +317,12 @@ llvm::StringRef SwiftREPL::GetSourceFileBasename() {
 bool SwiftREPL::SourceIsComplete(const std::string &source) {
   std::unique_ptr<llvm::MemoryBuffer> source_buffer_ap(
       llvm::MemoryBuffer::getMemBuffer(source));
+  swift::LangOptions empty_opts;
+  auto *swift_ast = getSwiftASTContext();
+  auto &lang_opts = swift_ast ? swift_ast->GetLanguageOptions() : empty_opts;
```

Comment:
I think it would be okay to just early return if there is no swift_ast, but this is fine, too.